BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/XFintoPUEO_Symmetric.py

```diff
@@ -0,0 +1,133 @@
+### File Name: XFintoPUEO.py
+### Created by: Jacob Weiler
+### Creation Date: 10/29/2022
+### Last Modified: 4/11/2023 
+### Description: This code is written to reformat the outputs of XF to be acceptable into PUEOsim. XF outputs one file, this code parses through that file and creates 6 new files. For information on inputs for PUEOsim check out this ELOG post: radiorm.physics.ohio-state.edu/elog/GENETIS/195 
```

Comment:
I think I just started doing this as a convention from classes I took (mostly in C++ I think). I think the main merit in having an author is to know who to ask for help (sometimes I put my email in there), but in practice that never happens anyway. I also think the file name is pretty redundant. 

I'm not familiar with module-level docstrings. I just looked it up--I didn't know that's what """ """ did! That can be useful, although we don't ever use these python scripts interactively or import them into different scripts (this latter point should probably be changed). But either way, it might be better for us to always change our block comments for script descriptions using dosctrings. I assume we should do this for functions too?